homepage/src/components/imageAndText.jsx

```diff
@@ -9,16 +12,16 @@ const ImageAndText = ({ id, image, title, subtitle, content, highlights }) => (
           <div className="image-and-text-details">
             <span className="name">{subtitle}</span>
             <h2 className="sub-position">{title}</h2>
-            <div
-              dangerouslySetInnerHTML={{
-                __html: content,
-              }}
-            />
+            <ReactMarkdown rehypePlugins={rehypeRaw}>{content}</ReactMarkdown>
```

Comment:
This logic applied everywhere. Please extract this component out to be an reusable component.

Reply:
done